build.sbt

```diff
@@ -1,53 +1,94 @@
-import EndpointsSettings._
+import xerial.sbt.Sonatype.GitHubHosting
 
-val `algebra-jvm` = LocalProject("algebraJVM")
-val `algebra-testkit-jvm` = LocalProject("algebra-testkitJVM")
-val `algebra-circe-testkit-jvm` = LocalProject("algebra-circe-testkitJVM")
-val `json-schema-generic-jvm` = LocalProject("json-schema-genericJVM")
-val `openapi-jvm` = LocalProject("openapiJVM")
+val algebra = "org.endpoints4s" %% "algebra" % "1.10.0"
+val algebraTestkit = "org.endpoints4s" %% "algebra-testkit" % "4.1.0"
```

Comment:
If that does not work with Scala 3, it should be possible to use the `Cross.for3Use2_13` trick, no?